signature/src/error.rs

```diff
@@ -83,18 +83,18 @@ impl From<Box<dyn core::error::Error + Send + Sync + 'static>> for Error {
     }
 }
 
-#[cfg(feature = "rand_core")]
-impl From<rand_core::Error> for Error {
-    #[cfg(not(feature = "alloc"))]
-    fn from(_source: rand_core::Error) -> Error {
-        Error::new()
-    }
+// #[cfg(feature = "rand_core")]
+// impl From<rand_core::OsError> for Error {
```

Comment:
I am not sure what we should do here. Whether we can create a Box<dyn Error> depends on a concrete RNG implementation. And we do not have specialization to work around this.

Reply:
I guess we can just get rid of them now that the concrete `rand_core::Error` type is gone

Reply:
We have `rand_core::OsError`, but it's used only by `OsRng`. The problem is that we can not do a generic conversion from an RNG error type since it's now depends on a concrete RNG and I am not sure how such change interacts with the rest of the `signature` code and its uses.